roiextractors/segmentationextractor.py

```diff
@@ -225,6 +250,7 @@ def get_num_rois(self):
         """
         return len(self.get_roi_ids())
 
+    @abstractmethod
     def get_images(self):
```

Comment:
this should be `raise NotImplementedError`, because there may be `SegmentationExtractors` that do not have summary images.

Reply:
most of them do have mean images, and for those that dont have, it will return None. 
In this branch it will return: {'Images': 'meanImg': None}}, 
again if the implementation (`self._images_mean`, `self._images_correlation`) in the caiman branch is good, then I can make those changes to all classes in this branch.